topostats/io.py

```diff
@@ -7,6 +7,7 @@
 from pathlib import Path
 import pickle as pkl
 from typing import Any, Dict, List, Union
+import regex
```

Comment:
This is failing on the GitHub Actions pipelines. If the regular expression works with the standard library then this could be replaced with...

```suggestion
import re
```

If its essential to use the [regex](https://pypi.org/project/regex/) package, which seems to offer the benefit of multi-threading, Unicode support, then we would have to add `regex` as a dependency under the `dependencies =[ ... ]` section of `pyproject.toml`.

I don't think there is anything that requires `regex` and if we can minimise the core package dependencies that is preferable, but you're probably more suited to know why `regex` has been used.

Reply:
I changed the library from regex to standard re.